### PR TITLE
Add wxMacMolPlt.app v7.7

### PR DESCRIPTION
**Pull request by jnooree**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).
- [x] `brew audit --cask {{cask_file}}` is error-free.
- [x] `brew style --fix {{cask_file}}` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew install --cask {{cask_file}}` worked successfully.
- [x] `brew uninstall --cask {{cask_file}}` worked successfully.

### CODE REVIEW

**Comment by miccal** [comment]
Thank you for your contribution to `homebrew-cask` @jnooree.

Could you please try to add a [`zap` stanza](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/zap.md), thanks.

**Comment by jnooree** [comment]
Added in jnooree/homebrew-cask@117605b!

**Comment by miccal** [comment]
@vitorgalvao and/or @reitermarkus thoughts on the `url` here?

File: Casks/wxmacmolplt.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+cask "wxmacmolplt" do
+  version "7.7"
+  sha256 :no_check
+
+  url "https://uofi.box.com/shared/static/ijfreodzydtqn214qui3snyvj5vdovqr.gz",
+      verified: "https://uofi.box.com/shared/static/"
```
Comment (reitermarkus) [inline Casks/wxmacmolplt.rb:6]: I'm not sure if the `uofi` part is enough to verify this. If possible the livecheck should also get the ID for the URL.

Comment (miccal) [reply Casks/wxmacmolplt.rb:6]: @reitermarkus I have added a `livecheck` for this, but there may be a better way?

Comment (reitermarkus) [reply Casks/wxmacmolplt.rb:6]: Thanks, @miccal. Kinda ugly, but it works. 😉

**Review by reitermarkus — APPROVED** [review]

**Pull request merged**